web/libs/editor/src/components/BottomBar/Controls.tsx

```diff
@@ -41,23 +41,31 @@ type ControlButtonProps = {
   onClick: (e: React.MouseEvent) => void;
 };
 
+export const EMPTY_SUBMIT_TOOLTIP = "Empty annotations denied in this project";
```

Comment:
minor note here - certainly non-blocking - but i see we're exporting it but we dont use it outside of this file
```suggestion
const EMPTY_SUBMIT_TOOLTIP = "Empty annotations denied in this project";
```